app/views/projects/bots/personal_access_tokens/_personal_access_token.html.erb

```diff
@@ -0,0 +1,43 @@
+<tr
+  id="<%= dom_id(personal_access_token) %>"
+  class="bg-white border-b dark:bg-slate-800 dark:border-slate-700"
+>
+  <td class="px-6 py-4"><%= personal_access_token.name %></td>
```

Comment:
**nitpick:** Can you update the table to show the count for tokens and then only show the names in the modal

Reply:
Updated in de455705bf183537e388c235f8b4d535b9deddf7